packages/svelte/test/helpers.js

```diff
@@ -200,7 +199,7 @@ export function create_loader(compileOptions, cwd) {
 				const fn = new AsyncFunction('__import', '__exports', transformed);
 				await fn(__import, __exports);
 			} catch (err) {
-				console.error({ transformed }); // eslint-disable-line no-console
+				console.error(compileOptions, transformed);
```

Comment:
I don't think this is valid? The first arg is a string and the second is supposed to be a string that gets substituted into the first

https://developer.mozilla.org/en-US/docs/Web/API/console/error

Reply:
It should match the other overload, `error(obj1, /* …, */ objN)`